TypeCobol/Compiler/Scanner/Scanner.cs

```diff
@@ -1561,12 +1557,23 @@ bool GuessIfCurrentTokenIsLevelNumber()
                                     return true;
 
                                 //Try to guess if it is a LevelNumber or Literal depending on previous tokens
-                                bool currentTokenIsExpectedToBeALiteral =
-                                    lastSignificantToken.TokenType == TokenType.OCCURS ||
-                                    lastSignificantToken.TokenType == TokenType.VALUE  ||
-                                    lastSignificantToken.TokenType == TokenType.VALUES ||
-                                    (beforeLastSignificantToken.TokenType == TokenType.VALUE && lastSignificantToken.TokenType == TokenType.IS) ||
-                                    (beforeLastSignificantToken.TokenType == TokenType.VALUES && lastSignificantToken.TokenType == TokenType.ARE);
+                                bool currentTokenIsExpectedToBeALiteral = false;
+                                switch (lastSignificantToken.TokenType)
+                                {
+                                    case TokenType.OCCURS:
+                                    case TokenType.VALUE:
+                                    case TokenType.VALUES:
+                                    case TokenType.THROUGH:
+                                    case TokenType.THRU:
+                                        currentTokenIsExpectedToBeALiteral = true;
+                                        break;
+                                    case TokenType.IS:
+                                    case TokenType.ARE:
```

Comment:
This PR now allow to use syntax `VALUE ARE` and `VALUES IS`.
Can you crate an issue about it and add an unit test?

Reply:
See issue #1782. Reverted to original code to separate between this PR and the "VALUES IS"/"VALUE ARE" issue.